core/css.js

```diff
@@ -434,6 +434,12 @@ Blockly.Css.CONTENT = [
     'cursor: url("<<<PATH>>>/handdelete.cur"), auto;',
   '}',
 
+  '.blocklyToolboxGrab {',
```

Comment:
New style added to distinguish between deletable and undeletable blocks over the toolbox.  Used in the block dragger.

CSS styles for the new comments are below.